packages/breadboard-web/src/index.ts

```diff
@@ -63,6 +65,10 @@ export class Main extends LitElement {
     if (!boardFromUrl) {
       return;
     }
+    const visualizer = currentUrl.searchParams.get("visualizer");
+    if (visualizer === "mermaid" || visualizer === "visualblocks") {
+      this.#visualizer = visualizer;
```

Comment:
nit: wonder if this is the point at which we inject the JS for the Visual Blocks? With it being ~5.8MB it seems a shame to load it on all occasions. What do you think?